.github/workflows/zap-scan.yml

```diff
@@ -0,0 +1,18 @@
+on:
+  schedule:
+    - cron:  '*/10 * * * *'
+
+jobs:
+  zap_scan:
+    runs-on: ubuntu-latest
+    name: Scan the webapplication
+    steps:
+      - name: Checkout
```

Comment:
We could remove this step, not really needed.

Reply:
We will be needing this step; when we are about to use the zap config file. As it is stored in the repository.

Shall I remove the checkout for this PR

Reply:
Right, better to leave it then.